src/gui/widgets/scroll_text.cpp

```diff
@@ -95,12 +105,7 @@ void scroll_text::set_text_alignment(const PangoAlignment text_alignment)
 
 void scroll_text::set_use_markup(bool use_markup)
 {
-	// Inherit.
 	styled_widget::set_use_markup(use_markup);
```

Comment:
Just remove this function now since its identical to the base class (see other comments).